ush/parsing_model_configure_FV3.sh

```diff
@@ -12,9 +12,9 @@
 
 FV3_model_configure(){
 
-export OUTPUT_FILETYPES="$OUTPUT_FILE"
+export OUTPUT_FILETYPES=${OUTPUT_FILETYPES:-${OUTPUT_FILE:-"netcdf"}}
 if [[ "$OUTPUT_FILE" == "netcdf" ]]; then
-   export OUTPUT_FILETYPES=" 'netcdf_parallel' 'netcdf' "
+   export OUTPUT_FILETYPES=${OUTPUT_FILETYPES:-" 'netcdf_parallel' 'netcdf' "}
```

Comment:
This will never run the way you have changed it. OUTPUT_FILETYPES is always already set, so it is never changed. Either this if-block needs to go, or something needs to be changed here.

Reply:
You're correct.  As-is, this if block is necessary for Jet which does not change `OUTPUT_FILETYPES` from `"netcdf"` in config.fcst.  Instead, I've deleted this if block and added a default setting in config.fcst.